README.md

```diff
@@ -86,11 +84,11 @@ High level stuff planned for Q1 2018 (January - March):
 |------------------|-------------------------------------|--------------|----------------|
 | tree             |      In-progress ([sneak peek][31]) |           -  |  [#3175][3175] |
```

Comment:
I think `tree` will it be released with `6.0.0` ?

Reply:
Yep